examples/jsm/controls/OrbitControls.js

```diff
@@ -25,6 +25,9 @@ class OrbitControls extends EventDispatcher {
 
 		super();
 
+		if ( domElement === undefined ) console.warn( 'THREE.OrbitControls: The second parameter "domElement" is now mandatory.' );
+		if ( domElement === document ) console.error( 'THREE.OrbitControls: "document" should not be used as the target "domElement". Please use "renderer.domElement" instead.' );
```

Comment:
Something is not right with this PR. You are adding unrelated changes. Consider to rebase your PR. Besides, please do not modify `examples/js/controls/OrbitControls.js`.